### PR TITLE
flamegraph 1.0 (new formula)

### PR DESCRIPTION
**Pull request by igorwwwwwwwwwwwwwwwwwwww**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Packaging FlameGraph for homebrew.

cc @brendangregg

### CODE REVIEW

File: Formula/flamegraph.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+class Flamegraph < Formula
+  desc "Stack trace visualizer"
+  homepage "https://github.com/brendangregg/FlameGraph"
+  url "https://github.com/brendangregg/FlameGraph/archive/v1.0.tar.gz"
+  sha256 "c5ba824228a4f7781336477015cb3b2d8178ffd86bccd5f51864ed52a5ad6675"
+  head "https://github.com/brendangregg/FlameGraph.git"
+
```
Comment (bayandin) [inline Formula/flamegraph.rb:7]: It looks like we don't need bottles

```suggestion

  bottle :unneeded

```

File: Formula/flamegraph.rb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  depends_on "perl"
```
Comment (bayandin) [inline Formula/flamegraph.rb:8]: Can we use default perl from macOS? 

If so, then
```suggestion
  uses_from_macos "perl"
```

**Review by chenrui333 — DISMISSED** [review]
LGTM

**Review by bayandin — DISMISSED** [review]

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
@bayandin Thanks, applied your suggestions! Let me know if you'd like me to sqash them into a single commit.

**Comment by bayandin** [comment]
> Let me know if you'd like me to sqash them into a single commit.

It would be great, Thanks!

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
@bayandin Done!

**Review by issyl0 — APPROVED** [review]
Thanks, Igor!

**Pull request closed without merging**